models/app.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = async () => {
+  const result = await db.query(`
+      SELECT *
+      FROM topics
+      `);
+  return result.rows;
```

Comment:
Nice use of Async Await! Careful with PSQL syntax, you are missing a ;

Reply:
Added ; in psql query